include/tnn_api.h

```diff
@@ -708,12 +710,11 @@ template <typename T_enc, typename T_net, int WIDTH> class NetworkWithEncodingMo
         this->sycl_queue_.wait();
 
         if (pack_gradient) {
-            net_gradients_->Packed(*net_gradients_.get());
+            net_gradients_->PackAndTranspose(*net_gradients_.get());
```

Comment:
This needs to be documented that there is not just packing but also transposing.

Reply:
documenting in modules.py where this is called

Reply:
also renaming to pack_transpose_gradient